use/adopter/whatsapp-flow-for-uci.md

```diff
@@ -0,0 +1,218 @@
+                "meta": {
+                    "body": "Hello ${name}-${phoneNo}, Netcore Test Notification Body demo dry run.",
+                    "type": "broadcast",
+                    "title": "Netcore Demo",
+                    "params": [
+                        "name",
+                        "phoneNo"
+                    ],
+                    "templateType": "JS_TEMPLATE_LITERALS"
+                }
+            }
+        ],
+        "adapter": "44a9df72-3d7a-4ece-94c5-98cf26307323"
+    }
+}'
+```
+
+This will create a conversation logic for the broadcast flow. We don't need an ODK form here as this is not a multi-message flow. The conversation logic for this type of bot only has a single broadcast message which is defined in the meta of the request.
+
+2. Create user segment:
+
+Creating a user segment is analogous to importing a list of users into UCI. The curl for this is given below.
+
+```bash
+curl --location 'http://localhost:3002/admin/user-segment' \
+--header 'admin-token: dR67yAkMAqW5P9xk6DDJnfn6KbD4EJFVpmPEjuZMq44jJGcj65' \
+--header 'Content-Type: application/json' \
+--data '{
+    "name": "Netcore demo",
+    "all": {
+        "type": "get",
+        "config": {
+            "url": "http://localhost:9080/service/testUserSegment2",
+            "type": "GET",
+            "cadence": {
+                "perPage": 1,
+                "retries": 5,
+                "timeout": 60,
+                "concurrent": true,
+                "pagination": true,
+                "concurrency": 10,
+                "retries-interval": 10
+            },
+            "pageParam": "page",
+            "credentials": {},
+            "totalRecords": 1
+        }
+    },
+    "byID": {
+        "type": "get",
+        "config": {
+            "url": "http://localhost:9080/service/testUserSegment2",
+            "type": "GET",
+            "cadence": {
+                "perPage": 1,
+                "retries": 5,
+                "timeout": 60,
+                "concurrent": true,
+                "pagination": true,
+                "concurrency": 10,
+                "retries-interval": 10
+            },
+            "pageParam": "page",
+            "credentials": {},
+            "totalRecords": 1
+        }
+    },
+    "byPhone": {
+        "type": "get",
+        "config": {
+            "url": "http://localhost:9080/service/testUserSegment2",
+            "type": "GET",
+            "cadence": {
+                "perPage": 1,
+                "retries": 5,
+                "timeout": 60,
+                "concurrent": true,
+                "pagination": true,
+                "concurrency": 10,
+                "retries-interval": 10
+            },
+            "pageParam": "page",
+            "credentials": {},
+            "totalRecords": 1
+        }
+    },
+    "count": 0
+}'
+```
+This creates a new user segment with the user list provided on `http://localhost:9080/service/testUserSegment2`.
```

Comment:
This seems a bit confusing as a user. If I have created a segment on external service already, then what is meant by 
`This creates a new user segment`. We need to be clearer, maybe write `This creates a config of segment data based on the user list...`.

Reply:
I think we should keep using the word `user segment` in this context rather than using `config` as we are referring it by the term using segment at a lot of places in the existing documentation. Replacing it with config might cause confusion at the adapter's end. Instead, we can refer to the hosted user data by the term `user list ' and make it clear that it should be provided by the adapter himself.